hedera-mirror-common/src/test/java/com/hedera/mirror/common/domain/DomainBuilder.java

```diff
@@ -749,6 +750,15 @@ public DomainWrapper<TransactionSignature, TransactionSignature.TransactionSigna
         return new DomainWrapperImpl<>(builder, builder::build);
     }
 
+    public DomainWrapper<UtilRandomGenerate, UtilRandomGenerate.UtilRandomGenerateBuilder> utilRandomGenerate() {
+        var builder = UtilRandomGenerate.builder()
+                .consensusTimestamp(timestamp())
+                .range(Integer.MAX_VALUE)
+                .pseudorandomBytes(bytes(384))
+                .pseudorandomNumber(random.nextInt());
```

Comment:
```suggestion
                .pseudorandomNumber(random.nextInt(Integer.MAX_VALUE));
```
when range > 0, the main nodes should generate a random integer in `[0, range)`